src/sagas/filtersAside.js

```diff
@@ -38,26 +42,27 @@ export function* fetchStat(action) {
 
     const requests = [];
 
+    filterParams = { ...defaultStatParams, ...filterParams };
     requests.push(call(Api.unitsStats, filterParams));
-    requests.push(call(Api.collectionsStats, filterParams));
-    requests.push(call(Api.labelsStats, filterParams));
+    countC && requests.push(call(Api.collectionsStats, filterParams));
+    countL && requests.push(call(Api.labelsStats, filterParams));
     if (isFilteredByBase) {
```

Comment:
![image](https://user-images.githubusercontent.com/1877419/174484251-534c5a70-b94c-45e3-a2d4-579ca1a1b765.png)

Reply:
I call it twice from component - one with isPrepare - true - with base params only. Its need for build tree of items - calls once. Second with filter params  - call every filter change